python/dgl/nn/pytorch/graph_transformer.py

```diff
@@ -216,6 +240,138 @@ def forward(self, ndata, attn_bias=None, attn_mask=None):
 
         attn = th.bmm(attn_weights, v_h).transpose(0, 1)
 
-        attn = self.out_proj(attn.reshape(N, bsz, self.feat_size).transpose(0, 1))
+        attn = self.out_proj(
+            attn.reshape(N, bsz, self.feat_size).transpose(0, 1)
+        )
 
         return attn
+
+
+class SparseBiasedMultiheadAttention(nn.Module):

```

Comment:
The name becomes sooooo long, not sure if there is a more concise alternative.

Reply:
How about replace `MultiheadAttention` in all module names with `MHA`?

Reply:
Sounds good to me, though you will need to update the other modules accordingly for consistency